deploy.go

```diff
@@ -676,6 +738,64 @@ func (gc *GatewayClient) CheckForCompletionQueueCommands(currentPhase string) (*
 	return &gatewayResponse, nil
 }
 
+// UninstallCluster used for start installation
+func (gc *GatewayClient) UninstallCluster(jsonStr, mdmUsername, mdmPassword, liaPassword string, allowNonSecureCommunicationWithMdm, allowNonSecureCommunicationWithLia, disableNonMgmtComponentsAuth, expansion bool) (*types.GatewayResponse, error) {
+
+	var gatewayResponse types.GatewayResponse
+
+	mapData, jsonParseError := jsonToMap(jsonStr)
```

Comment:
can this mapData be renamed to clusterData or clusterMap?

Reply:
Updated.